### PR TITLE
MultiPaste: Fix in_ids argument type in the schema

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
- It fixes a bug MultiPaste. An argument type was marked as int, when in fact a vector of int is expected.

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     *Changed from DALI_INT32 to DALI_INT_VEC*
 - Affected modules and functionalities:
     *MultiPaste*
 - Key points relevant for the review:
     *Check docs*
 - Validation and testing:
     *Existing tests apply*
 - Documentation (including examples):
     *Rendered docs should reflect that the argument expects a list of integers*

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Review by banasraf — APPROVED** [review]

File: dali/operators/image/paste/multipaste.cc [hunk 1]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
-.AddArg("in_ids", R"code(1D TensorList of type int.
+.AddArg("in_ids", R"code(Indices of the inputs to paste data from.)code",
+        DALI_INT_VEC, true)
+.AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
+of the source region.
 
-Indexes from what inputs to paste data in each iteration.)code", DALI_INT32, true)
-.AddOptionalArg<int>("in_anchors", R"code(2D TensorList of type int64
+2D data with the first dimension containing as many entries as elements in ``in_ids``
+and the second one containing as many elements as the number of dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:35]:
```suggestion
2D data with where first dimension corresponds to the elements of ``in_ids``
and the second one equal to the number dimensions of the data.
```
Dimension is a number, it doesn't contain anything.

File: dali/operators/image/paste/multipaste.cc [hunk 2]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
 
-Absolute values of LU corner of the selection for each iteration.
-Zeros are used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("shapes", R"code(2D TensorList of type int64
+If not provided, zeros are assumed.)code", nullptr, true)
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:37]:
```suggestion
If not provided, all anchors are zero.)code", nullptr, true)
```

File: dali/operators/image/paste/multipaste.cc [hunk 3]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
+.AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
 
-Absolute values of size of the selection for each iteration.
-Input size is used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("out_anchors", R"code(2D TensorList of type int64
+2D data with the first dimension containing as many entries as elements in ``in_ids``
+and the second one containing as many elements as the number of dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:41]:
```suggestion
2D data with where the first dimension corresponds to the elements of ``in_ids``
and the second one is equal to the number dimensions of the data.
```
Likewise.

File: dali/operators/image/paste/multipaste.cc [hunk 4]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
 
-Absolute values of LU corner of the paste for each iteration.
-Zeros are used if omitted.)code", nullptr, true)
+If not provided, the input shape is used.)code", nullptr, true)
+.AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
+of the destination region.
+
+2D data with the first dimension containing as many entries as elements in ``in_ids``
+and the second one containing as many elements as the number of dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:48]:
```suggestion
2D data with where the first dimension corresponds to the elements of ``in_ids``
and the second one is equal to the number dimensions of the data.
```
Likewise

File: dali/operators/image/paste/multipaste.cc [hunk 5]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
 Assumes HWC layout.)code")
-.AddArg("in_ids", R"code(1D TensorList of type int.
+.AddArg("in_ids", R"code(Indices of the inputs to paste data from.)code",
+        DALI_INT_VEC, true)
+.AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
+of the source region.
 
-Indexes from what inputs to paste data in each iteration.)code", DALI_INT32, true)
-.AddOptionalArg<int>("in_anchors", R"code(2D TensorList of type int64
+2D data with where the first dimension corresponds to the elements of ``in_ids``
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:34]:
```suggestion
2D data where the first dimension corresponds to the elements of ``in_ids``
```

File: dali/operators/image/paste/multipaste.cc [hunk 6]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
-Absolute values of LU corner of the selection for each iteration.
-Zeros are used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("shapes", R"code(2D TensorList of type int64
+If not provided, all anchors are zero.)code", nullptr, true)
+.AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
 
-Absolute values of size of the selection for each iteration.
-Input size is used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("out_anchors", R"code(2D TensorList of type int64
+2D data with where the first dimension corresponds to the elements of ``in_ids``
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:40]:
```suggestion
2D data where the first dimension corresponds to the elements of ``in_ids``
```

File: dali/operators/image/paste/multipaste.cc [hunk 7]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
-.AddArg("in_ids", R"code(1D TensorList of type int.
+.AddArg("in_ids", R"code(Indices of the inputs to paste data from.)code",
+        DALI_INT_VEC, true)
+.AddOptionalArg<int>("in_anchors", R"code(Absolute coordinates of LU corner
+of the source region.
 
-Indexes from what inputs to paste data in each iteration.)code", DALI_INT32, true)
-.AddOptionalArg<int>("in_anchors", R"code(2D TensorList of type int64
+2D data with where the first dimension corresponds to the elements of ``in_ids``
+and the second one is equal to the number dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:35]:
```suggestion
The anchors are represented as 2D tensors where the first dimension corresponds to the
elements of ``in_ids`` and the second one is equal to the number of dimensions of the
data, excluding channels.
```

File: dali/operators/image/paste/multipaste.cc [hunk 8]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
-Zeros are used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("shapes", R"code(2D TensorList of type int64
+If not provided, all anchors are zero.)code", nullptr, true)
+.AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
 
-Absolute values of size of the selection for each iteration.
-Input size is used if this is omitted.)code", nullptr, true)
-.AddOptionalArg<int>("out_anchors", R"code(2D TensorList of type int64
+2D data with where the first dimension corresponds to the elements of ``in_ids``
+and the second one is equal to the number dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:41]:
```suggestion
The shapes are represented as 2D tensors where the first dimension corresponds to the
elements of ``in_ids`` and the second one is equal to the number of dimensions of the
data, excluding channels.
```

File: dali/operators/image/paste/multipaste.cc [hunk 9]
```diff
@@ -26,23 +26,30 @@ This operator can also change the type of data.)code")
...
 
-Absolute values of LU corner of the paste for each iteration.
-Zeros are used if omitted.)code", nullptr, true)
+If not provided, the input shape is used.)code", nullptr, true)
+.AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
+of the destination region.
+
+2D data with where the first dimension corresponds to the elements of ``in_ids``
+and the second one is equal to the number dimensions of the data.
```
Comment (mzient) [inline dali/operators/image/paste/multipaste.cc:48]:
```suggestion
The anchors are represented as 2D tensors where the first dimension corresponds to the
elements of ``in_ids`` and the second one is equal to the number of dimensions of the
data, excluding channels.
```

**Review by mzient — APPROVED** [review]

**Comment by jantonguirao** [comment]
!build

**Pull request merged**